### PR TITLE
Clean up endpoint definitions in methods classes

### PR DESCRIPTION
**Pull request by PattaFeuFeu**
# Description

In this PR, I cleaned up how we refer to endpoints:

1. Ensure that all endpoints start _without_ a leading slash (`api/v1/…`, not `/api/v1/…`)
2. Ensure that all endpoints defined as a class-level `val` are named `endpoint` (or `endpointV1` and `endpointV2` if we have multiple) so that all method classes follow mostly the same structure
3. Ensure that all methods define their endpoints as class-level `val`s

Relates to #396

# Type of Change

- Refactoring and cleanup

# Breaking Changes

- None

# How Has This Been Tested?

I ran existing unit tests through `gradle check`

# Mandatory Checklist

- [x] I ran `gradle check` and there were no errors reported
- [x] I have performed a self-review of my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] All tests pass locally with my changes
- [ ] I have added KDoc documentation to all public methods

### CODE REVIEW

File: bigbone/src/test/kotlin/social/bigbone/api/method/DomainBlockMethodsTest.kt [hunk 1]
```diff
@@ -29,7 +29,7 @@ class DomainBlockMethodsTest {
 
         verify {
             client.get(
-                path = "/api/v1/domain_blocks",
+                path = "api/v1/domain_blocks",
```
Comment (andregasser) [inline bigbone/src/test/kotlin/social/bigbone/api/method/DomainBlockMethodsTest.kt:32]: Could we also introduce the `private val endpoint`constant for test classes?. I had a quick look and there are lots of places where we have duplicate endpoint strings.

**Review by andregasser — CHANGES_REQUESTED** [review]
Thanks for the cleanup 👍 . I had only one comment regarding test classes.

File: bigbone/src/test/kotlin/social/bigbone/api/method/DomainBlockMethodsTest.kt (hunk 1 above)
Comment (PattaFeuFeu) [reply bigbone/src/test/kotlin/social/bigbone/api/method/DomainBlockMethodsTest.kt:32]: I didn’t want to change that here and think we should first discuss in #396 which way to go with this. There are multiple options, some of which might be:
1. Using `internal` visibility for the endpoint `val`s
2. Introducing an endpoint `val` per test, like we do in the methods class under test
3. Having a central location for _all_ endpoints
4. …?

Comment (andregasser) [reply bigbone/src/test/kotlin/social/bigbone/api/method/DomainBlockMethodsTest.kt:32]: Ah ok, I did not check https://github.com/andregasser/bigbone/issues/396 yet. Fine for me.

**Review by andregasser — APPROVED** [review]
All good.

**Pull request merged**